src/universal/redux/AuthEngine.js

```diff
@@ -6,7 +6,7 @@ export default class AuthEngine {
     this.reducerName = reducerName;
   }
   saveToken(name, token, options, callback) {
-    this.store.dispatch(setAuthToken(token, this.reducerName));
+    this.store.dispatch(setAuthToken(token));
     if (callback) callback(null, token);
```

Comment:
Not sure why we need a callback if this is synchronous and we're just calling back with the same value that was provided to this function as a parameter.

Reply:
socket cluster API